sdk/opendp/whitenoise/evaluation/report/_report.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+import numpy as np
+
+class Report:
```

Comment:
How does this differ from the core library report?

Reply:
That's a good question. This one is agnostic to any specific DP implementation. All it needs is to pass the result of a PrivateQuery or DP function as a dataframe that we can then use for doing our evaluator checks. So quite lightweight. It allows for multiple dimensions (i.e. group by fields) and numerical columns within output to be tested in one-go.